cubic-server/world-storage/Level.cpp

```diff
@@ -0,0 +1,33 @@
+#include "Level.hpp"
+
+Level::Level()
+{
+}
+
+Level::~Level()
+{
+}
+
+ChunkColumn *Level::addChunkColumn(_2d_pos pos, ChunkColumn *chunkColumn) {
+    _chunkColumns.insert({pos, chunkColumn});
+    return _chunkColumns.at(pos);
```

Comment:
Instead of doing an `insert` with an `at` afterwards, `insert` already returns an iterator to the object inserted and the status of the insertion.

Reply:
do you want me to do something like that ?
```cpp
return _chunkColumns.insert({pos, chunkColumn}).first->second;
```